app/controllers/admin/users_controller.rb

```diff
@@ -1,9 +1,41 @@
-class Admin::UsersController < ApplicationController
+class Admin::UsersController < AdminController
+  before_action :load_user, except: %i(index new create)
   def new
     @user = User.new
   end
 
-  def create; end
+  def index
+    @users = User.where(role: params[:role]).page(params[:page])
```

Comment:
```suggestion
    @users = User.send(params[:role]).page(params[:page])
```
dc k? 
role là enum nhỉ

Reply:
role là enum ạ , nhưng không chạy khi dùng send ạ